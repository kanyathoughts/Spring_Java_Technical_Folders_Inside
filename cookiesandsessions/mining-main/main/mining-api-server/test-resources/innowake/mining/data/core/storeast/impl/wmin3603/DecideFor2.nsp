DEFINE DATA
LOCAL
1 #VAR1 (I4)
END-DEFINE

DECIDE
  FOR
    FIRST
      CONDITION
    WHEN #VAR1 = 5 AND #VAR1 EQ 4
		WRITE NOTITLE '1'
    WHEN #VAR1 = 15
    	IGNORE
    WHEN NONE 
		WRITE NOTITLE '1 NONE'
    WHEN ANY 
		WRITE NOTITLE '1 ANY'
END-DECIDE

END
