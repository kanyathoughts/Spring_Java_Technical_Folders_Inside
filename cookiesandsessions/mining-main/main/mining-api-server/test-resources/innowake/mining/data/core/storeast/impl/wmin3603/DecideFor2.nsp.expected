AstNode [type=ImpProgramObjectImpl, moduleId=1, location=AstNodeLocation [retracedOffset=0, retracedLength=280, assembledOffset=0, assembledLength=280, rootRelativeOffset=0, rootRelativeLength=280, rootRelativeStartLineNumber=1, rootRelativeEndLineNumber=20], inclusionLevel=-1, inclusionOrdinal=-1, hasAstModuleId=1]
 AstNode [type=DefineDataStmt, moduleId=1, location=AstNodeLocation [retracedOffset=0, retracedLength=44, assembledOffset=0, assembledLength=44, rootRelativeOffset=0, rootRelativeLength=44, rootRelativeStartLineNumber=1, rootRelativeEndLineNumber=4], inclusionLevel=-1, inclusionOrdinal=-1]
  AstNode [type=NaturalDataAreaNode, moduleId=1, location=AstNodeLocation [retracedOffset=13, retracedLength=19, assembledOffset=13, assembledLength=19, rootRelativeOffset=13, rootRelativeLength=19, rootRelativeStartLineNumber=2, rootRelativeEndLineNumber=3], inclusionLevel=-1, inclusionOrdinal=-1, properties=[dataAreaType=LOCAL]]
   AstNode [type=Variable, moduleId=1, location=AstNodeLocation [retracedOffset=22, retracedLength=10, assembledOffset=22, assembledLength=10, rootRelativeOffset=22, rootRelativeLength=10, rootRelativeStartLineNumber=3, rootRelativeEndLineNumber=3], inclusionLevel=-1, inclusionOrdinal=-1, properties=[byteLength=4, languageType=I, name=#VAR1, nameLength=5, nameOffset=0]]
    AstNode [type=Simple, moduleId=1, location=AstNodeLocation [retracedOffset=28, retracedLength=4, assembledOffset=28, assembledLength=4, rootRelativeOffset=28, rootRelativeLength=4, rootRelativeStartLineNumber=3, rootRelativeEndLineNumber=3], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=Integer, moduleId=1, location=AstNodeLocation [retracedOffset=29, retracedLength=2, assembledOffset=29, assembledLength=2, rootRelativeOffset=29, rootRelativeLength=2, rootRelativeStartLineNumber=3, rootRelativeEndLineNumber=3], inclusionLevel=-1, inclusionOrdinal=-1]
    AstNode [type=NotInitialized, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
 AstNode [type=DecideForStmt, moduleId=1, location=AstNodeLocation [retracedOffset=48, retracedLength=225, assembledOffset=48, assembledLength=225, rootRelativeOffset=48, rootRelativeLength=225, rootRelativeStartLineNumber=6, rootRelativeEndLineNumber=18], inclusionLevel=-1, inclusionOrdinal=-1, properties=[conditionPaths=[[1,0,0],[2,0,0]], LABEL_END_OFFSET=94, LABEL_START_OFFSET=48]]
  AstNode [type=First, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
  AstNode [type=ImpDecideForClauseImpl, moduleId=1, location=AstNodeLocation [retracedOffset=95, retracedLength=50, assembledOffset=95, assembledLength=50, rootRelativeOffset=95, rootRelativeLength=50, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=11], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=When, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
    AstNode [type=And, moduleId=1, location=AstNodeLocation [retracedOffset=100, retracedLength=24, assembledOffset=100, assembledLength=24, rootRelativeOffset=100, rootRelativeLength=24, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=Equal, moduleId=1, location=AstNodeLocation [retracedOffset=100, retracedLength=9, assembledOffset=100, assembledLength=9, rootRelativeOffset=100, rootRelativeLength=9, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
      AstNode [type=Ref, moduleId=1, location=AstNodeLocation [retracedOffset=100, retracedLength=5, assembledOffset=100, assembledLength=5, rootRelativeOffset=100, rootRelativeLength=5, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
       AstNode [type=Var, moduleId=1, location=AstNodeLocation [retracedOffset=100, retracedLength=5, assembledOffset=100, assembledLength=5, rootRelativeOffset=100, rootRelativeLength=5, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1, relations={ REFERS [to=Variable[#VAR1 (I4)]], BINDING [to=Variable[#VAR1 (I4)], label=definedBy] }, properties=[unresolvedFieldName=#VAR1]]
      AstNode [type=IntegerConstant, moduleId=1, location=AstNodeLocation [retracedOffset=108, retracedLength=1, assembledOffset=108, assembledLength=1, rootRelativeOffset=108, rootRelativeLength=1, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=Equal, moduleId=1, location=AstNodeLocation [retracedOffset=114, retracedLength=10, assembledOffset=114, assembledLength=10, rootRelativeOffset=114, rootRelativeLength=10, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
      AstNode [type=Ref, moduleId=1, location=AstNodeLocation [retracedOffset=114, retracedLength=5, assembledOffset=114, assembledLength=5, rootRelativeOffset=114, rootRelativeLength=5, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
       AstNode [type=Var, moduleId=1, location=AstNodeLocation [retracedOffset=114, retracedLength=5, assembledOffset=114, assembledLength=5, rootRelativeOffset=114, rootRelativeLength=5, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1, relations={ REFERS [to=Variable[#VAR1 (I4)]], BINDING [to=Variable[#VAR1 (I4)], label=definedBy] }, properties=[unresolvedFieldName=#VAR1]]
      AstNode [type=IntegerConstant, moduleId=1, location=AstNodeLocation [retracedOffset=123, retracedLength=1, assembledOffset=123, assembledLength=1, rootRelativeOffset=123, rootRelativeLength=1, rootRelativeStartLineNumber=10, rootRelativeEndLineNumber=10], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=WriteStmt, moduleId=1, location=AstNodeLocation [retracedOffset=128, retracedLength=17, assembledOffset=128, assembledLength=17, rootRelativeOffset=128, rootRelativeLength=17, rootRelativeStartLineNumber=11, rootRelativeEndLineNumber=11], inclusionLevel=-1, inclusionOrdinal=-1, properties=[uiStmtType=WRITE]]
    AstNode [type=Line, moduleId=1, location=AstNodeLocation [retracedOffset=142, retracedLength=3, assembledOffset=142, assembledLength=3, rootRelativeOffset=142, rootRelativeLength=3, rootRelativeStartLineNumber=11, rootRelativeEndLineNumber=11], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=InputText, moduleId=1, location=AstNodeLocation [retracedOffset=142, retracedLength=3, assembledOffset=142, assembledLength=3, rootRelativeOffset=142, rootRelativeLength=3, rootRelativeStartLineNumber=11, rootRelativeEndLineNumber=11], inclusionLevel=-1, inclusionOrdinal=-1]
      AstNode [type=StringConstant, moduleId=1, location=AstNodeLocation [retracedOffset=142, retracedLength=3, assembledOffset=142, assembledLength=3, rootRelativeOffset=142, rootRelativeLength=3, rootRelativeStartLineNumber=11, rootRelativeEndLineNumber=11], inclusionLevel=-1, inclusionOrdinal=-1]
  AstNode [type=ImpDecideForClauseImpl, moduleId=1, location=AstNodeLocation [retracedOffset=151, retracedLength=28, assembledOffset=151, assembledLength=28, rootRelativeOffset=151, rootRelativeLength=28, rootRelativeStartLineNumber=12, rootRelativeEndLineNumber=13], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=When, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
    AstNode [type=Equal, moduleId=1, location=AstNodeLocation [retracedOffset=156, retracedLength=10, assembledOffset=156, assembledLength=10, rootRelativeOffset=156, rootRelativeLength=10, rootRelativeStartLineNumber=12, rootRelativeEndLineNumber=12], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=Ref, moduleId=1, location=AstNodeLocation [retracedOffset=156, retracedLength=5, assembledOffset=156, assembledLength=5, rootRelativeOffset=156, rootRelativeLength=5, rootRelativeStartLineNumber=12, rootRelativeEndLineNumber=12], inclusionLevel=-1, inclusionOrdinal=-1]
      AstNode [type=Var, moduleId=1, location=AstNodeLocation [retracedOffset=156, retracedLength=5, assembledOffset=156, assembledLength=5, rootRelativeOffset=156, rootRelativeLength=5, rootRelativeStartLineNumber=12, rootRelativeEndLineNumber=12], inclusionLevel=-1, inclusionOrdinal=-1, relations={ REFERS [to=Variable[#VAR1 (I4)]], BINDING [to=Variable[#VAR1 (I4)], label=definedBy] }, properties=[unresolvedFieldName=#VAR1]]
     AstNode [type=IntegerConstant, moduleId=1, location=AstNodeLocation [retracedOffset=164, retracedLength=2, assembledOffset=164, assembledLength=2, rootRelativeOffset=164, rootRelativeLength=2, rootRelativeStartLineNumber=12, rootRelativeEndLineNumber=12], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=IgnoreStmt, moduleId=1, location=AstNodeLocation [retracedOffset=173, retracedLength=6, assembledOffset=173, assembledLength=6, rootRelativeOffset=173, rootRelativeLength=6, rootRelativeStartLineNumber=13, rootRelativeEndLineNumber=13], inclusionLevel=-1, inclusionOrdinal=-1]
  AstNode [type=ImpDecideForClauseImpl, moduleId=1, location=AstNodeLocation [retracedOffset=185, retracedLength=36, assembledOffset=185, assembledLength=36, rootRelativeOffset=185, rootRelativeLength=36, rootRelativeStartLineNumber=14, rootRelativeEndLineNumber=15], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=WhenNone, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=WriteStmt, moduleId=1, location=AstNodeLocation [retracedOffset=199, retracedLength=22, assembledOffset=199, assembledLength=22, rootRelativeOffset=199, rootRelativeLength=22, rootRelativeStartLineNumber=15, rootRelativeEndLineNumber=15], inclusionLevel=-1, inclusionOrdinal=-1, properties=[uiStmtType=WRITE]]
    AstNode [type=Line, moduleId=1, location=AstNodeLocation [retracedOffset=213, retracedLength=8, assembledOffset=213, assembledLength=8, rootRelativeOffset=213, rootRelativeLength=8, rootRelativeStartLineNumber=15, rootRelativeEndLineNumber=15], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=InputText, moduleId=1, location=AstNodeLocation [retracedOffset=213, retracedLength=8, assembledOffset=213, assembledLength=8, rootRelativeOffset=213, rootRelativeLength=8, rootRelativeStartLineNumber=15, rootRelativeEndLineNumber=15], inclusionLevel=-1, inclusionOrdinal=-1]
      AstNode [type=StringConstant, moduleId=1, location=AstNodeLocation [retracedOffset=213, retracedLength=8, assembledOffset=213, assembledLength=8, rootRelativeOffset=213, rootRelativeLength=8, rootRelativeStartLineNumber=15, rootRelativeEndLineNumber=15], inclusionLevel=-1, inclusionOrdinal=-1]
  AstNode [type=ImpDecideForClauseImpl, moduleId=1, location=AstNodeLocation [retracedOffset=227, retracedLength=34, assembledOffset=227, assembledLength=34, rootRelativeOffset=227, rootRelativeLength=34, rootRelativeStartLineNumber=16, rootRelativeEndLineNumber=17], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=WhenAny, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
   AstNode [type=WriteStmt, moduleId=1, location=AstNodeLocation [retracedOffset=240, retracedLength=21, assembledOffset=240, assembledLength=21, rootRelativeOffset=240, rootRelativeLength=21, rootRelativeStartLineNumber=17, rootRelativeEndLineNumber=17], inclusionLevel=-1, inclusionOrdinal=-1, properties=[uiStmtType=WRITE]]
    AstNode [type=Line, moduleId=1, location=AstNodeLocation [retracedOffset=254, retracedLength=7, assembledOffset=254, assembledLength=7, rootRelativeOffset=254, rootRelativeLength=7, rootRelativeStartLineNumber=17, rootRelativeEndLineNumber=17], inclusionLevel=-1, inclusionOrdinal=-1]
     AstNode [type=InputText, moduleId=1, location=AstNodeLocation [retracedOffset=254, retracedLength=7, assembledOffset=254, assembledLength=7, rootRelativeOffset=254, rootRelativeLength=7, rootRelativeStartLineNumber=17, rootRelativeEndLineNumber=17], inclusionLevel=-1, inclusionOrdinal=-1]
      AstNode [type=StringConstant, moduleId=1, location=AstNodeLocation [retracedOffset=254, retracedLength=7, assembledOffset=254, assembledLength=7, rootRelativeOffset=254, rootRelativeLength=7, rootRelativeStartLineNumber=17, rootRelativeEndLineNumber=17], inclusionLevel=-1, inclusionOrdinal=-1]
 AstNode [type=EndStmt, moduleId=1, location=AstNodeLocation [retracedOffset=277, retracedLength=3, assembledOffset=277, assembledLength=3, rootRelativeOffset=277, rootRelativeLength=3, rootRelativeStartLineNumber=20, rootRelativeEndLineNumber=20], inclusionLevel=-1, inclusionOrdinal=-1]
